src/components/AddWallet/AddWallet.jsx

```diff
@@ -46,13 +46,23 @@ export const AddWallet = () => {
               <FormLabel htmlFor="name">
                 {i18next.t('addWallet.name')}
               </FormLabel>
+
               <Input
-                {...register('name')}
+                {...register('name', {
+                  required: true,
+                  minLength: 2,
+                  maxLength: 30
+                })}
                 type="text"
                 placeholder={i18next.t('addWallet.name.placeholder')}
               />
               <FormErrorMessage>
-                {errors.name && errors.name.message}
+                {errors.name?.type === 'required' &&
+                  'Please, enter wallet name'}
+                {errors.name?.type === 'minLength' &&
+                  'Name must be longer than 1 symbols'}
+                {errors.name?.type === 'maxLength' &&
+                  'Name must be no longer than 30 symbols'}
```

Comment:
```suggestion
{() => {
   if(!error.name) return null;
  switch(errors.name?.type){
      case 'required': 
          return 'Please, enter wallet name';
          break;
     case 'maxLength': 
          return 'Name must be longer than 1 symbols';
          break;
     case 'minLength': 
          return 'Name must be no longer than 30 symbols';
          break;
  }
}}
```

but better move it to separate function

also error messages should be localised

Reply:
can I don't use break? 
 cause the return statement unconditionally exits a block of code, any statements after them cannot be executed.

Reply:
yep, but just make sure return statement is there

Reply:
great, and should i use a function with a currency error message too?

Reply:
https://react-hook-form.com/api/useform/register
You can pass all of this to react-hook-form and let it do the validating.

Reply:
Palina, I'm sorry, I don't get what you want me to change cause I thought I've already used react-hook-form for validation and somehow it validates input data.
My question was about the currency.error.message cause Anton made me change name.error.message, but not  currency.error.message